values-schema.yaml

```diff
@@ -614,6 +617,8 @@ properties:
         type: object
         additionalProperties: false
         properties:
+          _expert:
+            '$ref': '#/definitions/expertValues'
```

Comment:
`rawValues` would be better

Reply:
we use `raw` keyword when defining k8s resource, then let's not mix it with other things.

Reply:
doesn't matter, let's use rawValues

Reply:
"raw" means as is, so it has the same meaning

Reply:
done